observability/otel/otel-configuration-of-instrumentation.adoc

```diff
@@ -7,7 +7,6 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :FeatureName: OpenTelemetry instrumentation injection
```

Comment:
Note to self: removed because this is removed inhttps://github.com/openshift/openshift-docs/pull/84370.